src/deep-kernel/lib/ContainerAware.js

```diff
@@ -9,13 +9,15 @@ import {Kernel} from './Kernel';
 import {InvalidDeepIdentifierException} from './Exception/InvalidDeepIdentifierException';
 
 /**
+ * @todo - rename it to AbstractService (ContainerAware doesn't make sense anymore)
+ *
  * Container aware instance
  */
 export class ContainerAware extends MicroserviceInjectable {
```

Comment:
@AlexanderC pls create a separate issue for this, because it implies a lot of changes in other services